### ISSUE TITLE
Show more POI types in poi info (PP) subtitle

### ISSUE THREAD

**Issue by pastk** [organicmaps/organicmaps#8590]
#8481 has added most common types, but there are more.

power
![image](https://github.com/organicmaps/organicmaps/assets/18434508/ad6f8c4f-8524-417d-9fc8-bd68c0a73122)

piste:type
![image](https://github.com/organicmaps/organicmaps/assets/18434508/3a67fa5b-0506-470c-b64c-5518f049970b)

`shop` (generic) https://www.openstreetmap.org/way/355649602
![image](https://github.com/organicmaps/organicmaps/assets/18434508/23902a5d-8e7c-43ed-8723-538fb88f9c26)

**Comment by dvdmrtnz** [comment]
> `shop` (generic) https://www.openstreetmap.org/way/355649602
> ![image](https://private-user-images.githubusercontent.com/18434508/344325470-23902a5d-8e7c-43ed-8723-538fb88f9c26.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk2NTAyMjUsIm5iZiI6MTcxOTY0OTkyNSwicGF0aCI6Ii8xODQzNDUwOC8zNDQzMjU0NzAtMjM5MDJhNWQtOGU3Yy00M2VkLTg3MjMtNTM4ZmI4OGY5YzI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI5VDA4MzIwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIxNTQ4ZjkxYWM5YzZjMzAxNTc3ZGVkNTczYTU5MTNhNmZmMDJkNWM1MGZmZDk2NWE5OTA0MzM4MmQ3ZjFmMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.d2OFK2eW5XeTOU3HQVNzW98xOzKZb-FbnvKigna2evA)

I see. This is because we are filtering out level 1 types. @vng Any alternative idea on how to show level=1 `shop` but ignore level=1 `amenity` ?

https://github.com/organicmaps/organicmaps/blob/ab5eaced779a44938944f9ce0faa67eb19cbfab0/indexer/map_object.cpp#L109-L113

**Comment by biodranik** [comment]
The original generic amenity was amenity=clock: https://www.openstreetmap.org/node/489786064

I have a better idea than filtering the generic amenity. Let's leave it as is, and collect/fix user reports by adding missing amenities instead into OM. WDYT?

**Comment by pastk** [comment]
> 
> I see. This is because we are filtering out level 1 types. @vng Any alternative idea on how to show level=1 `shop` but ignore level=1 `amenity` ?

Make one of them an exception.

Or better make `shop=yes` transform into `shop-yes` (not generic `shop`).

> I have a better idea than filtering the generic amenity. Let's leave it as is, and collect/fix user reports by adding missing amenities instead into OM.

This is good, but it'll take a lot of time to add all missing tags (some of them might be niche / have very little use).

Probably it makes more sense not to match e.g. `amenity=clock` into `amenity` at all (but might break some legit cases, needs research).

**Comment by biodranik** [comment]
> This is good, but it'll take a lot of time to add all the missing tags (some of them might be niche / have very little use).

The goal is not to add _all_ tags (we may reach it someday eventually), but to add important/needed/useful tags (to users). Providing more detailed info on the map is one of the strongest features of OM/OSM.

Displaying "Clock" instead of "Amenity" is definitely more useful. And hacks are not needed.

**Comment by pastk** [comment]
> Displaying "Clock" instead of "Amenity" is definitely more useful. And hacks are not needed.

Sure! But do I understand right that the way you suggest to do it 

> I have a better idea than filtering the generic amenity. Let's leave it as is

is to continue to display generic "Amenity" if a specific full type is not supported yet?

**Comment by biodranik** [comment]
Yes. It will motivate us to add a new type based on discoveries and user reports )

**Comment by pastk** [comment]
> Yes. It will motivate us to add a new type based on discoveries and user reports )

But that also means that for types that we decided not to add (e.g. too little use) users will see a quite confusing "Amenity". In English it doesn't look too bad compared to translations to some other languages :)

**Comment by biodranik** [comment]
Essentially, in the perfect world, any type can be added, right?

**Comment by dvdmrtnz** [comment]
If you guys think we should still remove "Amenity", we could do something like this to detect generic amenity without matching amenity subtypes, which is clearer than previous code. And It will not remove generic shops for example.

```
auto const & isAmenity = ftypes::IsAmenityChecker::Instance();
if (isAmenity(type) && ftype::GetLevel(type) == 1))
  continue;
```

**Comment by pastk** [comment]
LGTM
